src/scripts/routes.js

```diff
@@ -0,0 +1,440 @@
+define([
+    "emby-button",
+    "emby-input",
+    "scripts/livetvcomponents",
+    "paper-icon-button-light",
+    "emby-itemscontainer",
+    "emby-collapse",
+    "emby-select",
+    "livetvcss",
+    "emby-checkbox",
+    "emby-slider",
+    "listViewStyle",
+    "dashboardcss"], function () {
+
+    function defineRoute(newRoute, dictionary) {
+        var baseRoute = Emby.Page.baseUrl();
+        var path = newRoute.path;
+        path = path.replace(baseRoute, "");
+        console.log("Defining route: " + path);
+        newRoute.dictionary = newRoute.dictionary || dictionary || "core";
+        Emby.Page.addRoute(path, newRoute);
+    }
+
+    console.log("Defining core routes");
+        defineRoute({
+            path: "/addplugin.html",
+            autoFocus: false,
+            roles: "admin",
+            controller: "addpluginpage"
+        });
+        defineRoute({
+            path: "/autoorganizelog.html",
+            roles: "admin"
+        });
+        defineRoute({
+            path: "/channelsettings.html",
+            autoFocus: false,
+            roles: "admin"
+        });
+        defineRoute({
+            path: "/addserver.html",
+            autoFocus: false,
+            anonymous: true,
+            startup: true,
+            controller: "addserver"
+        });
+        defineRoute({
+            path: "/dashboard.html",
+            autoFocus: false,
+            roles: "admin",
+            controller: "dashboardpage"
+        });
+        defineRoute({
+            path: "/dashboardgeneral.html",
+            controller: "dashboardgeneral",
+            autoFocus: false,
+            roles: "admin"
+        });
+        defineRoute({
+            path: "/dashboardhosting.html",
+            autoFocus: false,
+            roles: "admin",
+            controller: "dashboardhosting"
+        });
+        defineRoute({
+            path: "/devices/devices.html",
+            autoFocus: false,
+            roles: "admin",
+            controller: "devices"
+        });
+        defineRoute({
+            path: "/devices/device.html",
+            autoFocus: false,
+            roles: "admin",
+            controller: "device"
+        });
+        defineRoute({
+            path: "/dlnaprofile.html",
+            autoFocus: false,
+            roles: "admin"
+        });
+        defineRoute({
+            path: "/dlnaprofiles.html",
+            autoFocus: false,
+            roles: "admin"
+        });
+        defineRoute({
+            path: "/dlnaserversettings.html",
```

Comment:
what's the difference between `dlnaserversettings` and `dlnasettings`? can JF be a dlna client?..

Reply:
I don't know, i really didn't look into what each route does. Just kept the already made routes and general logic intact.

Reply:
Can you add a `// TODO: investigate` or something?

Reply:
Atually, dlnaserversettings route doesn't have a html file nor any usage in the app. So i will remove it.

Reply:
done